src/Domain/Tags/AreaElement.hpp

```diff
@@ -13,7 +13,8 @@ class DataVector;
 namespace domain::Tags {
 
 /*!
- * \brief The determinant of the induced Jacobian on a surface
+ * \brief The area element i.e. the determinant of the induced Jacobian on a
```

Comment:
Nitpicking, but I think better is "The area element, i.e. the determinant of the induced Jacobian, on a surface."